src/composables/maxAmount.ts

```diff
@@ -102,37 +102,34 @@ export function useMaxAmount({ store, formModel }: MaxAmountOptions) {
         );
         return;
       }
-
-      const minFee: BigNumber = new BigNumber(TxBuilder.calculateMinFee('spendTx', {
-        gas: sdk.Ae.defaults.gas,
-        params: {
-          ...sdk.Ae.defaults,
-          senderId: account.value.address,
-          recipientId: account.value.address,
-          amount: new BigNumber(val.amount > 0 ? val.amount : 0).shiftedBy(MAGNITUDE),
-          ttl: 0,
-          nonce: nonce.value + 1,
-          payload: val.payload,
-        },
-      })).shiftedBy(-MAGNITUDE);
-      if (!minFee.isEqualTo(fee.value)) fee.value = minFee;
+      const spendTx: Encoded.Transaction = await sdk.value.buildTx(Tag.SpendTx, {
+        senderId: account.value.address,
+        recipientId: account.value.address,
+        amount: BigNumber(val.amount > 0 ? val.amount : 0).shiftedBy(MAGNITUDE),
+        ttl: 0,
+        nonce: nonce.value + 1,
+        payload: '',
+      } as any); // TODO: Remove typecasting to any once https://github.com/aeternity/aepp-sdk-js/issues/1727 closed.
+      const { fee } = (unpackTx(spendTx, Tag.SpendTx) as any).tx; // TODO: Remove typecasting to any once https://github.com/aeternity/aepp-sdk-js/issues/1727 closed.
```

Comment:
```suggestion
      const { fee } = unpackTx(spendTx, Tag.SpendTx) as any; // TODO: Remove typecasting to any once https://github.com/aeternity/aepp-sdk-js/issues/1727 closed.
```
in the latest release `unpackTx` returns just transaction properties

Reply:
Resolving as this will be part of a separate PR